File: ELearningBackEnd/src/main/java/com/example/elearningbackend/note/NoteReq.java
package com.example.elearningbackend.note;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class NoteReq {

    private String title;

    private double duration;

    private String courseAlias;

    private long lectureId;
}

File: ELearningFrontEnd/src/app/dtos/note/note.req.ts
export class NoteReq{

    title:string;
    duration:number;
    courseAlias:string;
    lectureId:number;

    constructor(data: any) {
        this.title = data.title;
        this.duration = data.duration;
        this.courseAlias = data.courseAlias;
        this.lectureId = data.lectureId;

    }
}